docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,532 @@
+# Running BigDL-Orca Program on YARN
+
+This tutorial provides a step-by-step guide on how to run BigDL-Orca programs on Apache Hadoop (YARN) clusters, using a [PyTorch Fashin-MNIST program](https://github.com/intel-analytics/BigDL/tree/main/docs/docs/tutorials/tutorial_example/Fashion_MNIST/) as a working example.
+
+# 1. Key Concepts
+## 1.1 Init_orca_context
+A BigDL Orca program usually starts with the initialization of OrcaContext. For every BigDL Orca program, you should call `init_orca_context` at the beginning of the program as below:
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode, cores, memory, num_nodes, driver_cores, driver_memory, extra_python_lib, conf)
+```
+
+In `init_orca_context`, you may specify necessary runtime configurations for running the example on YARN, including:
+* `cluster_mode`: a String that specifies the underlying cluster; valid value includes `"local"`, `"yarn-client"`, `"yarn-cluster"`, `"k8s-client"`, `"k8s-cluster"`, `"bigdl-submit"`, `"spark-submit"`, etc.
```

Comment:
Remove etc? as you have listed all the values here.

Reply:
Also here we'd better highlight yarn for cluster mode, since it is a yarn tutorial.